### ISSUE TITLE
Would you mind sharing your implementation for the loss function you used during training?

### ISSUE THREAD

**Issue by yhymason** [ColorfulSoft/StyleTransfer-Colorization-SuperResolution#11]

**Comment by GlebSBrykin** [comment]
Hi, Haoyu Yang!
This repository contains codes that use already trained models posted by the authors in the public domain, so I did not train neural networks and did not implement the corresponding codes. However, you can find what you are interested in in the repositories of original implementations from the authors of the presented algorithms. Links to these repositories are available in the current repository in the descriptions for each algorithm.
Sincerely,
Gleb Brykin.
